github_tutorial.md

```diff
@@ -94,67 +97,77 @@ De beheerder van de standaard zal dus de volgende handelingen doen voor een vers
 	* Klik op "Propose new file"
 
 2. Ik wil een (tekst)document wijzigen
-	* Op en het document vanuit tab "Code" door te klikken op de bestandsnaam
+	* Ga naar de tab "Code"
+	* Kies daar de branch waarin je de wijzigingen wil aanbrengen
+	* Open het document dat je wil aanpassen door te klikken op de bestandsnaam
 	* Klik op het pen-icoon
-	* Voor de toevoegingen of wijzigingen door in de inhoud van het document
-	* Vul bij "Propose new file" een korte titel voor de toevoegingen of wijzigingen en een uitgebreide toelichting hiervan
-	* Klik op "Propose file change"
-
-3. Ik wil een opmerking maken over een bijdrage van iemand anders
+	* Voer de toevoegingen of wijzigingen door in de inhoud van het document
+	* Commit de wijzigingen in het document door op "Commit changes" te klikken waarbij je de radio button "Commit directly to the nnnnn branch." selecteert. "nnnnn" staat voor de branch waarin je deze wijziging wilde aanbrengen.
+
```

Comment:
Hierachter moet dan nog iets staan dat een pull request moet worden ingediend om het gewijzigde bestand naar de VNG-repository door te kunnen zetten.

Reply:
Dat had ik bewust niet gedaan omdat dit al in 'GitHub workflow (in de browser) voor open Standaarden' wordt genoemd. Het kan maar dan moet je dit consequent doortrekken in de andere use cases (waar van toepassing).